shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -173,6 +173,15 @@ - (NSDictionary*)getClipboardData:(NSString*)format;
  */
 - (void)setClipboardData:(NSDictionary*)data;
 
+/**
+ * Returns true iff the clipboard contains nonempty string data.
+ *
+ * See also:
+ *   * https://developer.apple.com/documentation/uikit/uipasteboard/1829416-hasstrings,
```

Comment:
I guess I didn't explain my earlier question very well: what information is this adding that helps understand the method? The description in the comment seems clear and unambiguous; following that link doesn't tell me anything new AFAICT, so why have it?

It also seems odd to document it in terms of another platform's implementation, given than a) we have a lot of platforms, so referring to one of them is pretty arbitrary, b) that implementation is not relevant here since there's no such method on macOS, and c) that anyone who wants to know how it's done on iOS can easily just go look at the iOS source.

Reply:
Sounds good, removed.